testing/tests/assert.in

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
I don't want to do this for every template, since the implementation for the `io::Write` adapter seems pretty simple and doesn't obviously intersect with other types of complexity in the templating engine. As such -- it would just slow down testing. We can add testing for it to a particular few test cases that you think are relevant.

As such, it doesn't seem like this macro adds much value.

Reply:
You can decide not to include the commit with the test changes.

Also if testing speed is an issue (which in fact isn't), we could make the `io::Write` test case execute only in 10% of the cases, chosen randomly.

Also if you compare the code before `assert_eq` and after `assert_render` you'll notice that the code is somewhat cleaner due to the removal of the `render().unwrap()`.